src/main/java/de/tum/in/www1/artemis/domain/ConflictingResult.java

```diff
@@ -0,0 +1,39 @@
+package de.tum.in.www1.artemis.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "conflicting-result")
+public class ConflictingResult {
+
+    /**
+     * id of model element in result that is in conflict
+     */
+    @Size(max = 50)
+    @Column(name = "modelElementId")
+    private String modelElementId;
+
+    @ManyToOne
+    private Result result;
+
+    /**
+     * feedback of modelElementId updated by the assessor during conflict resolution
+     */
+    private Feedback updatedFeedback;
```

Comment:
This would not save the association to Feedback in the database. The JavaDoc comment is too short. Please describe in more detail what exactly this association is used for!

Reply:
I updated the comment and added the missing @OneToOne association